insights/v1alpha1/types_insights.go

```diff
@@ -183,10 +244,12 @@ type InsightsReport struct {
 	// healthChecks provides basic information about active Insights health checks
 	// in a cluster.
 	// +listType=atomic
+	// +kubebuilder:validation:MaxItems=100
 	// +optional
 	HealthChecks []HealthCheck `json:"healthChecks,omitempty"`
 	// uri provides the URL link from which the report was downloaded.
 	// +kubebuilder:validation:Pattern=`^https:\/\/\S+`
+	// +kubebuilder:validation:MaxLength=2048
```

Comment:
Why 2048?

Reply:
I was not sure what length to put there, so I was searching in repo and I have found URL in `helm/v1beta1/types_project_helm_chart_repository.go` URL field that has length set to 2048 so I did the same. If you think it is way too much I can change it.